manifests/params.pp

```diff
@@ -51,4 +51,6 @@
   $rails_command='puma -w 5 -p $PORT --preload'
 
   $npm_timeout = 900
+
+  $pulp3 = false
```

Comment:
I'd default to true since every developer should have Pulp 3. @ehelms also suggested to stick to `pulpcore` instead of `pulp3`.